### DISCUSSION TITLE
ChunkLoadError

### DISCUSSION THREAD

**Discussion by MrAtlas** [vercel/next.js#59438] [Help]
### Summary

I was working on my app that i previously deployed and everything was working fine until I uploaded the new changes to github and redeployed and i keep getting this error, I really am not sure where it's coming from

#Uncaught SyntaxError: Identifier 'Audio' has already been declared

This is the first error, but I have no variable called Audio

#ChunkLoadError: Loading chunk 832 failed.
#Uncaught Error: Minified React error #423

And I am not sure what to do with these

this is the build url
https://phonesales-report-app-git-master-mratlas.vercel.app/

### Example

https://phonesales-report-app-git-master-mratlas.vercel.app/

**Comment by ankrgyl** [comment]
I'm hitting this as well. It seems to be because `openai` is being bundled twice (and hence redeclared)

**Reply by FunnyEngineer** [reply]
do you have a workaround for this?

**Reply by ankrgyl** [reply]
I think I solved this by globally pinning the openai version we use after running `pnpm update -r`
